examples/memory.py

```diff
@@ -22,7 +22,7 @@ def __init__(self):
         self.data = defaultdict(bytes)
         self.fd = 0
         now = time()
-        self.files['/'] = dict(st_mode=(S_IFDIR | 0o755), st_ctime=now,
+        self.files['/'] = dict(st_mode=(S_IFDIR | 0o777), st_ctime=now,
```

Comment:
is this needed?

Reply:
I believe it was needed because the POSIX and Windows permission models are very different. Will retest and see if it is really necessary.

Reply:
Thanks, let me know what you find. Otherwise I'll have to spin up a windows box.